v2/internal/frontend/desktop/linux/window.go

```diff
@@ -63,7 +74,10 @@ static void SetMinMaxSize(GtkWindow* window, int min_width, int min_height, int
     size.min_width = size.min_height = size.max_width = size.max_height = 0;
 
 	GdkRectangle monitorSize = getCurrentMonitorGeometry(window);
-    int flags = GDK_HINT_MAX_SIZE | GDK_HINT_MIN_SIZE;
+	if( isNULLRectangle(monitorSize) ) {
+		return;
```

Comment:
Just to make sure, do we still apply the constraints for a minimum or maximum in the `options.Options`?

IIRC the call to `SetMinMaxSize` for those is also very early and if the monitorSize couldn't be determined we would skip the constraints.

Reply:
I can't confirm tonight but it's interesting it worked at all with a null window. Potentially we'll need to hook on the "present"(?) Signal.

Reply:
Looks like the fix may cause issues with fullscreen.

Reply:
Can confirm min/max options do not get set on this branch.